pom.xml

```diff
@@ -82,9 +82,10 @@
 
     <modules>
         <module>deployment</module>
```

Comment:
I would move deployment to the bottom of the stack, just in case we do end up building artefacts in that module in future